core/libp2p/peer/address_repository/inmem_address_repository.hpp

```diff
@@ -0,0 +1,44 @@
+/**
+ * Copyright Soramitsu Co., Ltd. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#ifndef KAGOME_INMEM_ADDRESS_REPOSITORY_HPP
+#define KAGOME_INMEM_ADDRESS_REPOSITORY_HPP
+
+#include <list>
+#include <unordered_map>
+
+#include "libp2p/peer/address_repository.hpp"
+
+namespace libp2p::peer {
+
+  using Clock = std::chrono::steady_clock;
+
+  class InmemAddressRepository : public AddressRepository {
```

Comment:
Why is it not commented? I understand that its methods do the exact thing that is stated in the interface and thus need no comments, but the class itself is some particular implementation, which I think requires some explanation.

Reply:
fixed